UM/PackageManager.py

```diff
@@ -102,6 +102,56 @@ def _loadManagementData(self) -> None:
                 Logger.log("i", "User package management file %s doesn't exist, do nothing", self._user_package_management_file_path)
                 return
 
+        # For packages that become bundled in the new releases, but a lower version was installed previously, we need
+        # to remove the old lower version that's installed in the user's folder.
```

Comment:
Do we? Should the package manager not just load the most modern version of the two? I thought we spend quite some thought on this about a year ago to ensure that.

Reply:
Unfortunately, `PluginRegistry` doesn't do that.